<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1115</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1115</width>
    <height>630</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>方正细黑一简体</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>糖网病分级系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon_file/mainWindowIcon.png</normaloff>Icon_file/mainWindowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="MainTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1115</width>
      <height>590</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1115</width>
      <height>590</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1115</width>
      <height>590</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>方正细黑一简体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="testModeTab">
     <attribute name="title">
      <string>检测模式</string>
     </attribute>
     <widget class="QLabel" name="picViewLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>825</width>
        <height>550</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>825</width>
        <height>550</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>825</width>
        <height>550</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Welcome.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>20</width>
        <height>551</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QListView" name="picResourseListView">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>51</y>
        <width>251</width>
        <height>211</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>211</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>251</width>
        <height>211</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>310</y>
        <width>251</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正细黑一简体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>工具箱</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QSlider" name="scaleSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="blurBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/blur.png</normaloff>Icon_file/blur.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="brightBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/brightness.png</normaloff>Icon_file/brightness.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="contrastBtn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/contrast.png</normaloff>Icon_file/contrast.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="saturatBtn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/saturation.png</normaloff>Icon_file/saturation.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="scaleBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/scale.png</normaloff>Icon_file/scale.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="splitBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/splitchannel.png</normaloff>Icon_file/splitchannel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="waitBtn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon_file/wait.png</normaloff>Icon_file/wait.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="stateLabel">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>540</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正细黑一简体</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>欢迎使用...</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>10</y>
        <width>251</width>
        <height>35</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="loadPicBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>导入图片</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/load.png</normaloff>Icon_file/load.png</iconset>
         </property>
         <property name="shortcut">
          <string notr="true"/>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPicBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <stylestrategy>PreferDefault</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>清除图片</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/deleteAll.png</normaloff>Icon_file/deleteAll.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>270</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="checkPicBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>预览图片</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/show.png</normaloff>Icon_file/show.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deletePicBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>删除图片</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/delete.png</normaloff>Icon_file/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closePicBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>关闭预览</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/closeshow.png</normaloff>Icon_file/closeshow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>480</y>
        <width>251</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="analyseBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>开始分析</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/analyse.png</normaloff>Icon_file/analyse.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resultViewBtn">
         <property name="font">
          <font>
           <family>方正细黑一简体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>显示结果</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icon_file/playResult.png</normaloff>Icon_file/playResult.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="trainModeTab">
     <attribute name="title">
      <string>训练模式</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="font">
     <font>
      <family>方正细黑一简体</family>
     </font>
    </property>
    <property name="title">
     <string>&amp;文件</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
